easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,409 @@
+    }),
+    ('intreehooks', '1.0', {
+        'checksums': ['87e600d3b16b97ed219c078681260639e77ef5a17c0e0dbdd5a302f99b4e34e1'],
+    }),
+    ('pylev', '1.3.0', {
+        'checksums': ['063910098161199b81e453025653ec53556c1be7165a9b7c50be2f4d57eae1c3'],
+    }),
+    ('pastel', '0.2.1', {
+        'use_pip': False,
+        'checksums': ['e6581ac04e973cac858828c6202c1e1e81fee1dc7de7683f3e1ffe0bfd8a573d'],
+    }),
+    ('clikit', '0.6.2', {
+        'use_pip': False,
+        'checksums': ['442ee5db9a14120635c5990bcdbfe7c03ada5898291f0c802f77be71569ded59'],
+    }),
+    ('entrypoints', '0.3', {
+        'use_pip': False,
+        'checksums': ['c70dd71abe5a8c85e55e12c19bd91ccfeec11a6e99044204511f9ed547d48451'],
+    }),
+    ('tomlkit', '0.7.0', {
+        'use_pip': False,
+        'checksums': ['ac57f29693fab3e309ea789252fcce3061e19110085aa31af5446ca749325618'],
+    }),
+    ('shellingham', '1.3.2', {
+        'checksums': ['576c1982bea0ba82fb46c36feb951319d7f42214a82634233f58b40d858a751e'],
+    }),
+    ('requests-toolbelt', '0.9.1', {
+        'checksums': ['968089d4584ad4ad7c171454f0a5c6dac23971e9472521ea3b6d49d610aa6fc0'],
+    }),
+    ('pyrsistent', '0.16.1', {
+        'checksums': ['aa2ae1c2e496f4d6777f869ea5de7166a8ccb9c2e06ebcf6c7ff1b670c98c5ef'],
+    }),
+    ('pkginfo', '1.5.0.1', {
+        'checksums': ['7424f2c8511c186cd5424bbf31045b77435b37a8d604990b79d4e70d741148bb'],
+    }),
+    ('pexpect', '4.8.0', {
+        'checksums': ['fc65a43959d153d0114afe13997d439c22823a27cefceb5ff35c2178c6784c0c'],
+    }),
+    ('jsonschema', '3.2.0', {
+        'checksums': ['c8a85b28d377cc7737e46e2d9f2b4f44ee3c0e1deac6bf46ddefc7187d30797a'],
+    }),
+    ('simplejson', '3.17.2', {
+        'checksums': ['75ecc79f26d99222a084fbdd1ce5aad3ac3a8bd535cd9059528452da38b68841'],
+    }),
+    ('webencodings', '0.5.1', {
+        'checksums': ['b36a1c245f2d304965eb4e0a82848379241dc04b865afcc4aab16748587e1923'],
+    }),
+    ('html5lib', '1.1', {
+        'checksums': ['b2e5b40261e20f354d198eae92afc10d750afb487ed5e50f9c4eaf07c184146f'],
+    }),
+    ('cleo', '0.8.1', {
+        'use_pip': False,
+        'checksums': ['3d0e22d30117851b45970b6c14aca4ab0b18b1b53c8af57bed13208147e4069f'],
+    }),
+    ('cachy', '0.3.0', {
+        'checksums': ['186581f4ceb42a0bbe040c407da73c14092379b1e4c0e327fdb72ae4a9b269b1'],
+    }),
+    ('msgpack', '1.0.0', {
+        'checksums': ['9534d5cc480d4aff720233411a1f765be90885750b07df772380b34c10ecb5c0'],
+    }),
+    ('CacheControl', '0.12.6', {
+        'checksums': ['be9aa45477a134aee56c8fac518627e1154df063e85f67d4f83ce0ccc23688e8'],
+    }),
+    ('ptyprocess', '0.6.0', {
+        'use_pip': False,
+        'checksums': ['923f299cc5ad920c68f2bc0bc98b75b9f838b93b599941a6b63ddbc2476394c0'],
+    }),
+    ('lockfile', '0.12.2', {
+        'checksums': ['6aed02de03cba24efabcd600b30540140634fc06cfa603822d508d5361e9f799'],
+    }),
+    ('poetry-core', '1.0.0', {
```

Comment:
@Micket `poetry` itself is missing?

Reply:
So, poetry -> keyring -> secretstorage -> hidden dependency on jeepney which is python3 only (and has only ever supported python3).  But, since it seems to be only optionally used, I added back secretstorage and disabled the check.